t/json.t

```diff
@@ -8,24 +8,40 @@ my $mysql = Mojo::mysql->new($ENV{TEST_ONLINE});
 my $db    = $mysql->db;
 
 eval {
-  $db->query('create table if not exists mojo_json_test (id int(10), name varchar(60), j json)');
```

Comment:
I don't think you can move this out of the eval block, since I want to skip the test if the mysql/mariadb version does not support JSON.

Reply:
I've replaced it with a call to `json_type` because this is much faster than trying to create a table. My assumption was that a DB that has no json support doesn't understand this command. At least on MySQL 5.6 it works.

Reply:
Works on MariaDB 10.1 too.